### PR TITLE
Here are some old ASL guidelines in md.

### PR DESCRIPTION
**Pull request by aringler-usgs**
These may or may not be valuable.  If they are I can certainly clean them up more.  

I have almost certainly messed up my pull request (apologies in advance) let me know if you want it done differently.

### CODE REVIEW

File: ASLGuidelines.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+*Include error handling:* Try to include error handling to avoid people using code that would mask problems.  For example, make sure variables format correctly and throw exceptions when things go wrong.  
+
+*Avoid making special versions of code:* If someone wants a special version of a piece of code try to make it a feature in the current code.  Try to avoid making spin-off versions that cause later issues.
+
+*Minimize Warnings:* Warnings should be treated as errors. Code committed should not contain any errors.  When possible, avoid producing code that produces different warnings on specific compilers.  This can be hard to keep track of and sometimes is inconsistent do your best to make your code workable.
+
+### Releases
+
+In order to avoid a lot of head scratching aim to release your code with a versioning scheme.  The use of three numbers separated by decimals makes logical sense.  The first number indicates the major version, the second number indicates the minor version, and the third number indicates minor bug fixes.
```
Comment (isuftin) [inline ASLGuidelines.md]: Semantic Versioning! http://semver.org/

Comment (emartinez-usgs) [inline ASLGuidelines.md]: :+1:

Comment (aringler-usgs) [inline ASLGuidelines.md]: Cheers for that!  Looks like we reinvented the wheel.  Thanks.

**Comment by llastowka-usgs** [comment]
Thank you for your contribution @aringler-usgs ! And i like that idea @emartinez-usgs, we could move @mguy-usgs review guidelines to a separate file too.

**Pull request merged**